kafka-scheduler/src/test/java/com/mesosphere/dcos/kafka/scheduler/KafkaSchedulerTest.java

```diff
@@ -82,6 +86,42 @@ public void testRescheduleOneTask() {
     }
 
     @Test
+    public void testResourceOffersSuppress() throws Exception {
+        kafkaScheduler = new KafkaScheduler(kafkaSchedulerConfiguration, environment) {
+            // Create install StageManager with no operations
+            @Override
+            protected StageManager createInstallStageManager(Stage installStage, PhaseStrategyFactory strategyFactory) {
+                StageManager stageManager = super.createInstallStageManager(installStage, strategyFactory);
+                StageManager stageManagerSpy = spy(stageManager);
+                Stage stage = stageManagerSpy.getStage();
+                Stage stageSpy = spy(stage);
+                when(stageSpy.isComplete()).thenReturn(true);
+                when(stageManagerSpy.getStage()).thenReturn(stageSpy);
+                return stageManagerSpy;
+            }
+
+            // Create RecoveryScheduler with no operations
+            @Override
+            protected DefaultRecoveryScheduler createRecoveryScheduler(KafkaOfferRequirementProvider offerRequirementProvider) {
+                DefaultRecoveryScheduler scheduler = super.createRecoveryScheduler(offerRequirementProvider);
+                DefaultRecoveryScheduler spy = spy(scheduler);
+                when(spy.hasOperations(null)).thenReturn(false);
+                return spy;
+            }
+        };
+
+        kafkaScheduler.resourceOffers(driver, Collections.emptyList());
+        verify(driver, times(1)).suppressOffers();
+    }
+
+
+    @Test
+    public void testStatusUpdateRevive() throws Exception {
+        kafkaScheduler.statusUpdate(driver, getTestTaskStatus());
+        verify(driver, times(1)).reviveOffers();
+    }
+
```

Comment:
The existing recovery tests already test this.  That is, if the integration test I wrote to verify the suppressed state passes, and the recovery test passes, then we know kafka successfully transitioned from suppressed -> revived.